src/pocketmine/utils/Random.php

```diff
@@ -100,7 +100,7 @@ public function nextSignedInt(){
 		$this->y = $this->z;
 		$this->z = $this->w;
 		$this->w = ($this->w ^ (($this->w >> 19) & 0x7fffffff)
-							 ^ ($t ^ (($t >> 8) & 0x7fffffff))) & 0xffffffff;
```

Comment:
This is necessary because tabs may have different size on different editors, and using spaces for inline indentation is more appropriate.